curl_cffi/requests/websockets.py

```diff
@@ -97,13 +142,13 @@ def loop(self):
             self._loop = asyncio.get_running_loop()
         return self._loop
 
-    async def arecv(self):
+    async def arecv(self) -> Tuple[bytes, int]:
         return await self.loop.run_in_executor(None, self.recv)
 
     async def asend(self, payload: bytes, flags: CurlWsFlag = CurlWsFlag.BINARY):
         return await self.loop.run_in_executor(None, self.send, payload, flags)
 
-    async def aclose(self):
-        await self.loop.run_in_executor(None, self.close)
+    async def aclose(self, code: int = WsCloseCode.OK, message: bytes = b""):
```

Comment:
```suggestion
    async def aclose(self, code: WsCloseCode = WsCloseCode.OK, message: bytes = b""):
```

Reply:
Close codes outside of that enum are allowed to be used (i.e. 3000-4999, which are not defined by the spec)